charts/ccd-data-store-api/values.preview.template.yaml

```diff
@@ -0,0 +1,5 @@
+java:
+  image: ${IMAGE_NAME}
+  ingressHost: ${SERVICE_FQDN}
+  postgresql:
```

Comment:
This should be at root level, outside java.

Reply:
why's that? I assumed since its a dependency of the java app it would go inside of it

Reply:
https://github.com/helm/helm/issues/4490#issuecomment-469423429 . Limitation from Helm